package.json

```diff
@@ -28,5 +28,10 @@
     "prettier": "prettier --config \"./.prettierrc.yml\" --write \"**/*.{json,md,sol,yml}\"",
     "prettier:check": "prettier --config \"./.prettierrc.yml\" --check \"**/*.{json,md,sol,yml}\"",
     "solhint": "solhint --config \"./.solhint.json\" \"{src,test}/**/*.sol\""
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "forge snapshot --check"
```

Comment:
do you need to specify this here? `yarn lint-staged` works fine in the pre-commit file alone

Reply:
`yarn lint-staged` isn't checking for gas snapshot, so I added it

Reply:
you can remove this section, and it will still work fine. adding hooks in both `package.json` and in `.husky/pre-commit` is unnecessary

the package.json addition was only needed in earlier versions: https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v8